lib/logger.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Utility functions for command line output logging from Protractor.
+ * May be used in different processes, since the launcher spawns
+ * child processes for each runner.
+ *
+ * All logging directly from Protractor, its driver providers, or runners,
+ * should go through this file so that it can be customized.
+ */
+export module Logger {
```

Comment:
this looks like old TypeScript syntax for a "internal module" which was replaced by namespace in https://github.com/Microsoft/TypeScript/issues/2159

https://basarat.gitbooks.io/typescript/content/docs/project/namespaces.html